source/Note.hx

```diff
@@ -88,6 +88,9 @@ class Note extends FlxSprite
 					ignoreNote = mustPress;
 					reloadNote('HURT');
 					noteSplashTexture = 'HURTnoteSplashes';
+					if (PlayState.isPixelStage) {
+						noteSplashTexture = 'HURTnoteSplashes-pixel';
+				   }
```

Comment:
Messed-up indentation.

Reply:
bruh, i'll try to fix it

Reply:
i'll need to close the pr and open another, cuz i've lost the repo and the notesplashes had been moved to preload, it is just for some update